org/ystia/yorc/yorc/linux/ansible/playbooks/configure.yml

```diff
@@ -46,3 +46,14 @@
       file:
         path: /tmp/yorc.d/
         state: absent
+
+    - name: "Ensures {{ DATA_DIR }}/.ssh dir exists"
+      file:
+        path: "{{ DATA_DIR }}/.ssh"
+        state: directory
+
+    - name: "Copy key pair"
+      copy:
+        content: "{{ SSH_KEY }}"
+        dest: "{{ DATA_DIR }}/.ssh/yorc.pem"
+        mode: 0400
```

Comment:
Missing owner yorc to .ssh and yorc.pem key. Got a permission denied while trying to access the key when I deploy an app.